packages/components/src/button/index.native.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { TouchableOpacity } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+export function Button( props ) {
+	const { onClick } = props;
```

Comment:
You can also use deconstructing with `aria-label`:
```js
const { children, onClick, 'aria-label': ariaLabel } = props;
```
and include `children` props, too.